devops/openshift/redis-ha-deploy.yml

```diff
@@ -0,0 +1,213 @@
+                  memory: ${MEMORY_REQUEST}
+                limits:
+                  cpu: ${CPU_LIMIT}
+                  memory: ${MEMORY_LIMIT}
+              volumeMounts:
+                - name: conf
+                  mountPath: /conf
+                  readOnly: false
+                - name: data
+                  mountPath: /data
+                  readOnly: false
+                - name: script
+                  mountPath: /script
+                  readOnly: false
+          volumes:
+            - name: conf
+              secret:
+                secretName: ${NAME}-creds
+            - name: script
+              configMap:
+                name: ${NAME}
+                defaultMode: 0755
+      volumeClaimTemplates:
+        - metadata:
+            name: data
+            labels:
+              app: ${NAME}
+          spec:
+            accessModes:
+              - ReadWriteOnce
+            resources:
+              requests:
+                storage: ${PVC_SIZE}
+          storageClassName: ${STORAGE_CLASS}
+parameters:
+  - name: NAME
+    description: The name of the application for labelling all artifacts.
+    displayName: Application Name
+    required: true
+    value: redis
+  - name: REPLICAS
+    description: The number of StatefulSet replicas to use.
+    displayName: Replicas
+    required: true
+    value: "6"
+  - name: IMAGE_REGISTRY
+    description: The base docker registry
+    displayName: Docker Image Registry
+    required: true
+  - name: IMAGE_NAME
+    description: Redis Image Name
+    displayName: Image Name
+    required: true
+  - name: IMAGE_TAG
+    description: Redis Image Tag
+    displayName: Image Tag
+    required: true
+  - name: CPU_REQUEST
+    description: Starting amount of CPU the container can use.
+    displayName: CPU Request
+    required: true
+    value: 25m
+  - name: CPU_LIMIT
+    description: Maximum amount of CPU the container can use.
+    displayName: CPU Limit
+    required: true
+    value: 100m
+  - name: MEMORY_REQUEST
+    description: Starting amount of memory the container can use.
+    displayName: Memory Request
+    required: true
+    value: 256Mi
+  - name: MEMORY_LIMIT
+    description: Maximum amount of memory the container can use.
+    displayName: Memory Limit
+    required: true
+    value: 256Mi
+  - name: PVC_SIZE
+    description: The size of the persistent volume to create.
+    displayName: Persistent Volume Size
+    required: true
+    value: 1Gi
+  - name: STORAGE_CLASS
+    description: The type of the persistent volume to create.
+    displayName: Persistent Volume Class
+    required: true
+    value: netapp-file-standard
+  - name: REDIS_PORT
+    description: Redis port.
+    displayName: Redis client port.
+    required: true
+    value: "6379"
+  - name: REDIS_BUS_PORT
+    description: Redis cluster bus port.
+    displayName: Redis cluster bus port used for node to node communication.
+    required: true
+    value: "16379"
```

Comment:
Is the "1" supposed to be here or should it be "6379"?

Reply:
This is the bus port which is generally 1000 + redis client port so 16379.